### PR TITLE
landing page: add citations search box

### PR DESCRIPTION
**Pull request by jennur**
Closes https://github.com/zenodo/zenodo-rdm/issues/135

## Screenshot
Screenshot 2022-12-06 at 11 26 47

### CODE REVIEW

**Comment by kpsherva** [comment]
looks good!, only a small comment, see: https://github.com/zenodo/zenodo-rdm/issues/135#issuecomment-1339268343

File: assets/less/zenodo-rdm/globals/site.overrides [hunk 1]
```diff
@@ -106,3 +106,6 @@ a.inverted {
   height: 0.85em;
   width: 1.2em;
 }
+.uppercase {
+  text-transform: uppercase !important;
```
Comment (kpsherva) [inline assets/less/zenodo-rdm/globals/site.overrides:110]: invenio-theme candidate?

Comment (zzacharo) [reply assets/less/zenodo-rdm/globals/site.overrides:110]: I wouldn't put it if that's only for this case as I think is a hack to uppercase the values...but indeed seems like a generic class to add in invenio-theme!

**Comment by jennur** [comment]
## Update
- Had to move the accordion to React to get the counter in at the panel title.

The accordion from basic (html, jQuery) SUI vs React SUI behaves differently, as the React one apparently does not have a transition option like the basic one does. If it's not super important, (for now) maybe we can leave it as is? I'm not sure how to solve this issue nicely atm, so please lmk if you have suggestions.

Screenshot 2022-12-06 at 15 22 25

**Pull request merged**